jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -128,11 +134,11 @@ class MatchBoxZoom(_MatchedZoomMixin, BoxZoom):
 
 
 @viewer_tool
-class MatchPanZoom(_MatchedZoomMixin, ImagePanZoom):
+class MatchImagePanZoom(_MatchedZoomMixin, ImagePanZoom):
     icon = os.path.join(ICON_DIR, 'panzoom_match.svg')
     tool_id = 'jdaviz:panzoommatch'
```

Comment:
@kecnry , I didn't rename the `tool_id` because people do access the tool using that ID, or at least we tell them that in the docs. FYI.

Reply:
ok.  If it's just in the development docs, I think it would be ok to change, I wouldn't consider that public API since we don't have any downstream packages.  But either way, I think the `tool_id` and class name should be somewhat consistent to avoid confusion for us later (so if you don't want to change the previous `tool_id`, then maybe the new class should be `SimpleMatchPanZoom`).  I'll leave that decision to you though and review the rest of the PR.

Reply:
OK I changed it back.